### PR TITLE
Pptp 1874

### PR DESCRIPTION
**Pull request by ajchilts**
### Description of Work carried through

Changes to add amend return endpoint for submission of amended return to EIS.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.

### CODE REVIEW

**Comment by ajchilts** [comment]
// TODO do we need to delete the submitted return as above when successful?

**Review by liampan — APPROVED** [review]
that todo otherwise looks gtm

File: app/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/ReturnsSubmissionController.scala [hunk 1]
```diff
@@ -61,6 +62,17 @@ class ReturnsSubmissionController @Inject() (
       }
     }
 
+  // TODO do we need to delete the submitted return as above when successful?
```
[the remark this answers is not in the document]
Comment (lmcewanhmrc) [reply app/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/ReturnsSubmissionController.scala:65]: Yeah. once accepted by etmp, we should wipe the in-flight return from the mongo cache I reckon

**Comment by liampan** [comment]
Will look at todo in future pr now.

**Pull request merged**